README.md

```diff
@@ -97,6 +97,9 @@ representing the user choice (resolved on confirmation and rejected on cancellat
 | **`allowClose`**                        | `boolean`   | `true`                  | Whether natural close (escape or backdrop click) should close the dialog. When set to `false` force the user to either cancel or confirm explicitly.                                                                                   |
 | **`confirmationKeyword`**               | `string`    | `undefined`             | If provided the confirmation button will be disabled by default and an additional textfield will be rendered. The confirmation button will only be enabled when the contents of the textfield match the value of `confirmationKeyword` |
 | **`confirmationKeywordTextFieldProps`** | `object`    | `{}`                    | Material-UI [TextField](https://mui.com/material-ui/api/text-field/) props for the confirmation keyword textfield.                                                                                                                     |
+| **`acknowledgement`**               | `boolean &#124; string`    | `false`             | If `true`, it shows the acknowledge checkbox with default label `Please confirm` and disables the confirm button while the checkbox is unchecked. If `string` it will show this string as checkbox label. |
```

Comment:
Let's make it always a string (default to `undefined`), it's a rather specific use case, so I think usually it's going to be a specific text as well :)